#This message is used to send command to tetra
#Parameter read command

int32 num
---
int32 P0_UART1_BAUDRATE
int32 P1_UART2_BAUDRATE
int32 P2_SONAR_SELECT
int32 P3_SONAR_DISTANCE_OFFSET
int32 P4_SONAR_QUANTITY
int32 P5_SONAR_MAX_DISTANCE
int32 P6_NC
int32 P7_SONAR_NONE_DETECT_MODE
int32 P8_BATTERY_CHARGING_OFFSET
int32 P9_BATTERY_CHARGING_GAIN
int32 P10_STATUS_LED_MIN
int32 P11_STATUS_LED_MAX
int32 P12_BATTERY_LED_MIN
int32 P13_BATTERY_LED_MAX
int32 P14_LED_CH1_OFFSET
int32 P15_LED_CH2_OFFSET
int32 P16_LED_CH3_OFFSET
int32 P17_LED_CH4_OFFSET
int32 P18_LED_CH5_OFFSET
int32 P19_LED_CH6_OFFSET
int32 P20_LED_CH7_OFFSET
int32 P21_LED_CH8_OFFSET
int32 P22_POWER_ENABLE
int32 P23_OUTPORT
int32 P24_BATTERY_RECHARGE_VOLTAGE
int32 P25_BATTERY_RECHARGE_OFFSET
int32 P26_BATTERY_MIN
int32 P27_BATTERY_MAX
int32 P28_BATTERY_SAMPLING_TIME
int32 P29_CONVEYOR_MODE
int32 P30_CONVEYOR_TERMINAL_BASE
int32 P31_CONVEYOR_TERMINAL_OFFSET
bool command_Result

